resources/views/cruds/datagrids/filters/datagrid-filter.blade.php

```diff
@@ -42,4 +43,8 @@
     <x-dialog id="datagrid-filters" :loading="true" full="true">
 
     </x-dialog>
+
+    <x-dialog id="datagrid-bookmark" :loading="true" full="true">
```

Comment:
No need for another dialog, you can reuse the one that is already on every single page